lib/xcodeproj/project.rb

```diff
@@ -318,11 +320,27 @@ def pretty_print
     #
     def save(save_path = nil)
       save_path ||= path
+      @dirty = false if save_path == path
```

Comment:
if saving to the path this project was read from, the project will no longer be dirtied compared to what's on disk after the save